pyccel/ast/builtin_methods/list_methods.py

```diff
@@ -16,10 +16,11 @@
 
 __all__ = ('ListAppend',
            'ListClear',
-           'ListPop'
+           'ListInsert',
+           'ListPop',
            )
 
-
+#=======================================================================================
```

Comment:
Please try to keep the lines within 80 characters (including the terminal newline characters).

Reply:
fixed in 12cd0b2a08f4f7d59e7c61e94769a7a9d9092039